index.html

```diff
@@ -4,10 +4,133 @@
+                <li class="nav-item"><a class = "nav-link contact-link" href="#">Publications</a></li>
+                <li class="nav-item"><a class = "nav-link contact-link" href="#">Donate</a></li>
     
+            </ul>
+    </nav>
+            </nav>
+    <div class="home-page-content">
+        <div>
+            <div class="welcome-text">
+                <P>"Hello! Poetic Souls"</P>
+                <h1>
+                   Slam Poetry Jam
+                    2022
+                </h1>
+
+            </div>
+            <div class="event-messege">
+                <p>A joyful celebration of Poetry and Story.Join us and Listen to celebreted  poets read their work and discuss what they do !</p>
+            </div>
+            <div class="event-info">
+                <p>
+                    2022.08.01(FRI) ~ 02(sat)
+                </p>
+                <p>
+                @ Times Square 7 pm UTC+5,
+                </p>
+
+            </div>
+            </div>
+        </div>
+    </div>
+<section class="program-info-page">
+        <div class="program">
+            <div class="main-program"><p>Main Program</p>
+            <hr>
+        </div>
+        <div class="program-info-wrap">
+        </div>
+         <div class="btn" >
+                <button class="join-btn">Join The Slam Poetry Event </button>
+                <a href="#">See The Whole Program </a>
+                
+            </div>
+        </div>
+        
+ </section>
+
+ <section class="featured-speaker" >
+     <div>
+            <div class="main-program">
+            <p class="featured-speakers-txt">Featured Speakers</p>
+            <hr>
+        </div>
+        <div class="speaker-list-wrap">
+            </div>
+        </div>
+        <div class="mobile-speaker-more">
+            <p>MORE</p>
+            <i class="fa-solid fa-chevron-down"></i>
+        </div>
+ </section>
+
+
+<section class="sponsers">
+     <div class="partner">
+       
+            <div class="partner-txt">
+                <p>Partner</p></div>
+            <div>
+                <hr class="sml-line-br">
+            </div>
+            <div class="partner-img">
+                <img src="./src/partner-img-apple.png" alt="Apple Logo">
+                <img src="./src/partner-img-pepsi.png" alt="Pepsi Logo">
+                <img src="./src/partner-img-twitter.png" alt="twitter Logo">
+                <img src="./src/partner-img-cocacola.png" alt="CocaCola Logo">
+                <img src="./src/partner-img-macdonald.png" alt="MAcdonal Logo">
+        </div>
+        
+    </div>
+</section>
+ <footer>
+    <img class="logo" src="./src/poetry-title-img.png" alt="Poetry Logo">
+    <div>
+        <p>2020 Poetry Slam .Some Right Reserved</p>
+        <p>WaterBrook Multnomah
+10807 New Allegiance Drive, Suite 500
+Colorado Springs, CO 80921
+United States
++1-719-590-4999</p>
+    </div>
+ </footer>
```

Comment:
- [ ] [OPTIONAL] The footer does not display at all on the homepage(desktop). I am only pointing this out because the previous reviewer didn't mention it but I strongly suggest you work on it.